ClimsoftVer4/ClimsoftVer4/ucrNavigation.vb

```diff
@@ -14,6 +14,7 @@
         iCurrRow = 0
         iMaxRows = dtbRecords.Rows.Count
         If strSortCol <> "" AndAlso dtbRecords.Columns.Contains(strSortCol) Then
+            'TODO. This won't work, probably do the sorting at the EF level
```

Comment:
Why doesn't this work?

Reply:
@dannyparsons.  Because our datatable has no idea about the actual column types in the EF. We designed our code to assume that all columns are of type string. Therefore the sorting that is done gives unexpected results sometimes. For instance check on the wrong output when you sort any of our selectors.
Something like;
1
2
3
10
4
30

after sorting would result to;

1
10
2
3
30
4

I would have either made the datatable "aware" of the column types or implemented sorting at the EF level. But I'm looking forward to what you and David will come up with.